instrumentserver/apps.py

```diff
@@ -83,6 +90,9 @@ def serverScript() -> None:
         tempFile.close()
         Path(stationConfig).unlink(missing_ok=True)
 
+def quitPollingThread():
```

Comment:
add a '_' at the beginning of the function name. This indicates that this is a "private" method and users should not use this function. Should look like `_quitPollingThread`